
Processing:   0%|                                                                                                                                            | 0/30 [00:00<?, ?it/s]
tensor([0, 0, 1, 1, 1, 1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0])
Processing:   0%|                                                                                                                                            | 0/30 [00:10<?, ?it/s]
Traceback (most recent call last):
  File "/Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/classifier/main.py", line 62, in <module>
    main()
  File "/Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/classifier/main.py", line 54, in main
    model_conv = train_model(run, model_conv, config.model_name, dataloaders, criterion, optimizer_conv,
  File "/Users/lihongxuan/Desktop/AIPI/Courses/AIPI540/AIPI-540/classifier/utils/train.py", line 73, in train_model
    y_trues[i*dataloaders[phase].batch_size:(i*dataloaders[phase].batch_size + inputs.size(0))] = labels
RuntimeError: The expanded size of the tensor (1) must match the existing size (16) at non-singleton dimension 1.  Target sizes: [16, 1].  Tensor sizes: [16]